climada/hazard/wildfire.py

```diff
@@ -0,0 +1,1165 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Define WildFire class.
+"""
+
+__all__ = ['WildFire']
+
+import logging
+import warnings
+import itertools
+from datetime import date
+from functools import partial
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy import sparse
+from sklearn.neighbors import BallTree
+from sklearn.cluster import DBSCAN
+import numba
+import geopandas as gpd
+from shapely.geometry import Point
+from shapely.ops import transform
+import pyproj
+
+from climada.hazard.centroids.centr import Centroids
+from climada.hazard.base import Hazard
+from climada.hazard.tag import Tag as TagHazard
+from climada.util.constants import ONE_LAT_KM, DEF_CRS
+from climada.util.dates_times import str_to_date
+
+from climada.util.alpha_shape import alpha_shape, plot_polygon
+warnings.simplefilter(action='ignore', category=FutureWarning)
```

Comment:
Why this line? Why ignore warnings?

Reply:
I've removed the alpha_shape and the hull_bruned_area method in commit 13a2c8c as calculating area burned is not used and the module is complex already